.prettierrc.json

```diff
@@ -2,6 +2,5 @@
   "semi": false,
   "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": "none",
```

Comment:
It looks like the vast majority of changes are related to this config change. What's the motivation? Just asking in case we can keep this to a less extensive set of changes.

Reply:
Staying consistent with the other repos. Since the plan was to eventually unify all their configs by having them use the shareable configs created in https://github.com/reduxjs/redux-toolkit/pull/4138, I thought taking care of line endings/formatting issues in one PR would make things less unpredictable so you guys won't have to deal with a bunch of formatting changes in an unrelated PR in case somebody decides to run Prettier before pushing their changes.